designer/uix/kivy_console.py

```diff
@@ -721,10 +721,10 @@ def run_cmd(*args):
             else:
                 try:
                     command = re.sub('[ ]+', ' ', command)
-                    if command[3] == os.sep:
+                    if os.path.isdir(command[3]):
```

Comment:
this is always false. command will be something like `cd /home/aron/test`. This `if` is checking if I'm moving to a sub-folder or to a root-folder.